### ISSUE TITLE
The first video explainers, and a list that cannot tell a 26-minute video from a blog post

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#389]
`check:explainers` has had a full oEmbed verification path for `kind: video` since ADR-0017 — 404 for a dead video, title **and** author compared against what YouTube reports — and it has never run against a single entry. All 482 explainers are `read`.

The finder from #384/#386 produced candidates; 21 were verified through oEmbed on a laptop. **Eight of them map to a primary domain** and can land now, reaching **81 concepts**:

| domain | concepts | |
|---|---|---|
| Alignment | 14 | AssemblyAI — RLAIF vs. RLHF: the technology behind Anthropic's Claude |
| Attention | 13 | 3Blue1Brown — Attention in transformers, step-by-step |
| Reinforcement | 13 | Google DeepMind — RL Course by David Silver, Lecture 2 |
| Vision | 9 | DeepFindr — Vision Transformer Quick Guide |
| Landscape | 8 | 3Blue1Brown — Gradient descent, how neural networks learn |
| Continual | 8 | Paul Hand — Continual Learning and Catastrophic Forgetting |
| Information | 8 | Adian Liusie — Intuitively Understanding the KL Divergence |
| Graphs | 8 | WelcomeAIOverlords — Simple Message Passing on Graphs |

The other 13 are blocked on #388 and are not part of this.

## The rendering problem this creates

The sources list already carries the heading **"Watch & read"**, so the design anticipated videos — but nothing in the markup distinguishes one:

```astro
<a href={e.url}>{e.title}</a> — {e.author}
{e.scope === 'domain' && <span> · on {node.domain[0]}</span>}
```

A 26-minute lecture and a blog post render identically. Until now that was harmless, because every entry was a `read`. With the first videos landing it stops being harmless: the reader clicks expecting to skim and gets a video, or the reverse. CLAUDE.md §4 requires that meaning never rest on a single ambiguous carrier, and here there is no carrier at all.

This is a consequence of the content change rather than an unrelated tidy-up, so it belongs in the same PR.

## Acceptance criteria

- [ ] Eight domain-scoped video explainers added, attached by `domain[0]`, reaching 81 concepts
- [ ] They are **additions**, not replacements — every affected node keeps its existing `read`, so a reader gets both a thing to read and a thing to watch
- [ ] A video is marked as such in the list, in text and not by colour alone
- [ ] `check:explainers` verifies all 21 videos' titles and authors against oEmbed in CI — the path exercised for the first time
- [ ] `validate:content`, the golden fixture, and the perf budget all unaffected

Refs #61, #365, #384, #388
